test/helpers.js

```diff
@@ -136,6 +140,152 @@ const Helpers = {
     for (let i = 0; i < results.length; i++) {
       console.log(`slot #${i}`, results[i]);
     }
+  },
+  initHelperArtifacts(_artifacts) {
```

Comment:
I think this is a wrong place for contract deployment scripts. It makes Helpers logic dirty. Should be moved to the separate file.

Reply:
TODO added